util/reggen/register.py

```diff
@@ -652,3 +653,39 @@ def apply_alias(self, alias_reg: 'Register', where: str) -> None:
 
             # Validate and override attributes.
             field.apply_alias(alias_field, where)
+
+    def scrub_alias(self, where: str) -> None:
+        '''Replaces sensitive fields in register with generic names
+
+        This function can be used to create the generic register descriptions
+        from full alias hjson definitions. It will only work on registers
+        where the alias_target keys are defined, and otherwise throw an error.
+        '''
+        # These attributes are scrubbed
+        assert (self.alias_target)
```

Comment:
Nit: These parens are not needed. Also, it's probably better to make this `assert self.alias_target is not None` to avoid weirdness to do with empty strings.

Reply:
Done!